data/json/items/gun/223.json

```diff
@@ -426,6 +426,16 @@
       }
     ]
   },
+  {
+    "id": "m38dmr",
+    "copy-from": "m27iar",
+    "name": { "str": "M38 DMR" },
+    "type": "GUN",
+    "description": "The M38 Designated Marksman Rifle is an M27 IAR, itself derived from a H&K416, selected for accuracy, and outfitted with a variable power scope and a QDSS suppressor.",
+    "dispersion": 125,
+    "price_postapoc": 3000,
+    "built_in_mods": [ "suppressor", "rifle_scope" ]
```

Comment:
Shouldn't those be default mods rather than built-in mods? So that they're detachable, but the gun still spawns with them.

Reply:
Ah, yes, you're right. My mistake. Thank you for noticing :-)